CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # MicroProfile Language Server Changelog
 
+## [0.13.0](https://github.com/eclipse/lsp4mp/milestone/14?closed=1) (October 17, 2024)
+
+### Enhancements
+
+- feat: Provide property validator extension API. See [#460](https://github.com/eclipse/lsp4mp/pull/460).
+
+### Bug Fixes
+
```

Comment:
It seems there's two other bug fixes:
 - https://github.com/eclipse/lsp4mp/commit/18b7076558f66b905a2fb877db078fa413ab7327
 - https://github.com/eclipse/lsp4mp/commit/b7540636149dd76a874abedf0c6f1ee090b8e3ac

Reply:
Good catch!

For https://github.com/eclipse/lsp4mp/commit/b7540636149dd76a874abedf0c6f1ee090b8e3ac itwas a bug that I introduced in this millestion, so we need to skip it

Reply:
fixed